### PR TITLE
Use `[ReplaceCulture]` to get tests running with different time formats

### PR DESCRIPTION
**Pull request by dougbu**
- #2160

nits:
- auto-remove trailing whitespace
- let VS do its thing w.r.t. test services in an .xproj file

### CODE REVIEW

**Comment by maumar** [comment]
:shipit:

**Comment by dougbu** [comment]
b6118c9

File: test/EntityFramework.Sqlite.FunctionalTests/project.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "dependencies": {
         "EntityFramework.Relational.FunctionalTests": "7.0.0-*",
-        "EntityFramework.Sqlite": "7.0.0-*"
+        "EntityFramework.Sqlite": "7.0.0-*",
+        "Microsoft.AspNet.Testing": "1.0.0-*"
```
Comment (bricelam) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: NOOOOOOOOOOOOOO!!!!!!!!!!

Comment (bricelam) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: (We'll discuss taking a dependency on this as a team. It's fine for now.)

Comment (ajcvickers) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: We just had a mini-discussion in the team room and decided we should discuss as a team whether or not this is a good dependency to take. //cc @divega

Comment (divega) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: We talked and the conclusion is that it is fine for now, but we can remove it if it causes problems in the future.

Comment (ErikEJ) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: Please dont!

Comment (divega) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: Is it because you don't have the package and you'll need it now?

Comment (ErikEJ) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: @divega  I have not tried, but having this dependency may pull in other ASP.NET dependencies, and it seems irrelevant for functional tests of a desktop only provider.

Comment (MichaelKetting) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: @dougbu I can't seem to properly restore the dependency on "Microsoft.AspNet.Testing" when I'm compiling via Visual Studio or using "dnu restore" from the command line. It might be related to a similar issue that exists in the "EntityFramework.Sqlite" project after a dependency on "System.IO" was added. I've mentioned this over in https://github.com/aspnet/EntityFramework/pull/2218.

@ErikEJ The ReplaceCulture attribute is needed to harden the tests against formatting changes when running them on a system that isn't using the US system culture settings. It's also an replacement for this PR: https://github.com/aspnet/EntityFramework/pull/2194.

Comment (ErikEJ) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: @MichaelKetting Thanks, Yes, I understand that, but I would imagine that it could be fixed in the implementaion of the Relational parameter logger code.

Comment (MichaelKetting) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: @ErikEJ Certainly. And this was one of the two ways on how to fix it. @rowanmiller suggested that it should be fixed in the test, i.e. the logger output should follow the current culture (see https://github.com/aspnet/EntityFramework/issues/2160).

Comment (ErikEJ) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: Just did a "rebel" PR https://github.com/aspnet/EntityFramework/pull/2328

Comment (MichaelKetting) [inline test/EntityFramework.Sqlite.FunctionalTests/project.json]: @ErikEJ :smile:

**Comment by dougbu** [comment]
>  I can't seem to properly restore the dependency on "Microsoft.AspNet.Testing"

@MichaelKetting `Microsoft.AspNet.Testing` unfortunately diverged over time from providing a few core / all-platform test helpers. @bricelam's aspnet/Testing#119 and #2326 fixes will get things working again with both .sln files. (Not absolutely sure #2326 is required for this in particular but it definitely looks positive.)

**Comment by dougbu** [comment]
b6118c9

**Comment by ErikEJ** [comment]
Looks less and less like a "good" solution to a simple test issue!
